### DISCUSSION TITLE
Is there a way to change the mqtt topic for electricity power?

### DISCUSSION THREAD

**Discussion by brainrecursion** [Arnold-n/P1P2MQTT#151] [Q&A] [answered]
Is there a way to use a different topic for electricity power, as I already have a different mqtt topic which publishes the real time energy power in watts for the heat pump?

**Comment by Arnold-n** [comment, marked as answer]
Yes, command `P35 <topic>` followed by `D5` to save permanently, command can be given via telnet or via topic `P1P2/W`.

**Reply by brainrecursion** [reply]
Thanks that worked well.
